MIGRATING-0.6-0.7.md

```diff
@@ -1,168 +1,89 @@
 # Migrating from embedded-graphics 0.6.x to 0.7.0
 
+> Please note that this migration guide may be incomplete in some sections. If any missing or incorrect information is found, please [open an issue](https://github.com/embedded-graphics/embedded-graphics/issues/new) or [join the Matrix chatroom](https://matrix.to/#/#rust-embedded-graphics:matrix.org) to bring it to our attention.
+
 ## Table of contents
 
 - [Migrating from embedded-graphics 0.6.x to 0.7.0](#migrating-from-embedded-graphics-06x-to-070)
   - [Table of contents](#table-of-contents)
```

Comment:
This is autogenerated by whatever VSCode plugin I'm using unfortunately. I could remove that and manually update the ToC but it would be nice (and reduce mistakes) to have it generated. Would you strongly prefer to remove these items?

Reply:
I would prefer it, but not enough to warrant any extra manual work. Just leave the items in the TOC, if there is no option in the plugin to remove them.

Reply:
I found some options to remove the top level heading and the recursive item for the ToC so thankfully this is sorted now.